MMM-MBTA.js

```diff
@@ -92,170 +106,283 @@ Module.register("MMM-MBTA", {
+                        break;
+                    case "6": // Gondola case
+                        // There shouldn't be a gondola in Boston.
+                    case "7": // Funicular case
+                        // There shouldn't be a funicular in Boston.
+                    default:
+                        symbolCell.className = "fa fa-question-circle-o";
+                }
 
-            // ETA
-            if (this.config.showETATime) {
-                var preAwayCell = document.createElement("td");
-                var preAwayTime = parseInt(this.stationData[i].preAway);
-                if (preAwayTime < 10) { // Better to display single digits as "now"
-                    preAwayCell.innerHTML = "Now";
-                } else {
-                    var minutes = Math.floor(preAwayTime / 60);
-                    var seconds = preAwayTime % 60;
+                // Color Icons
+                if (this.config.colorIcons) {
+                    switch (this.stationData[i].routeId) {
+                        case "Red":
+                        case "Mattapan":
+                            symbolCell.className += " red";
+                            break;
+                        case "Blue":
+                            symbolCell.className += " blue";
+                            break;
+                        case "Orange":
+                            symbolCell.className += " orange";
+                            break;
+                        case "Green-B":
+                        case "Green-C":
+                        case "Green-D":
+                        case "Green-E":
+                            symbolCell.className += " green";
+                            break;
+                        case "Boat-F1":
+                        case "Boat-F4":
+                            symbolCell.className += " green";
+                            break;
+                    }
 
-                    if (this.config.showMinutesOnly) {
-                        if (minutes === 0){
-                            preAwayCell.innerHTML = "< 1";
-                        } else {
-                            preAwayCell.innerHTML = minutes;
-                        }
-                    } else if (this.config.formatETA) { // Parses the time away into MM:SS
-                        // Padding so we don't get something like 4:3 minutes...
-                        if (seconds < 10) {
-                            // lol what even is type casting
-                            seconds = "0" + seconds;
-                        }
+                    if (this.stationData[i].routeId.includes("CR-")) {
+                        symbolCell.className += " commuter"
+                    }
 
-                        preAwayCell.innerHTML = minutes + ":" + seconds;
-                    } else {
-                        preAwayCell.innerHTML = seconds;
+                    if (isNaN(this.stationData[i].routeId) === false) {
```

Comment:
Same thing here, can just be
```js
if (this.stationData[i].routeId)  { ... }
```

Reply:
What do you mean? Only bus routes use straight numbers. Everything else is alphanumeric.

I can replace it with `if ($.isNumeric(this.stationData[i].routeId))` instead so it's a little cleaner and clearer. Checking whether something is not _not_ a number is kinda dumb.